cypress-tests/cypress/support/commands.js

```diff
@@ -763,30 +781,31 @@ Cypress.Commands.add(
 Cypress.Commands.add(
   "createPaymentIntentTest",
   (
-    request,
+    createPaymentBody,
```

Comment:
Can we please maintain a better naming convention going forward and that is `requestBody` instead?

Reply:
CreatePaymentBody is understandable right instead of requestBody. We get to know from which fixtures it getting

Reply:
Okay, can you please make relevant changes to all of them? Possibly can be taken up as a separate PR focusing only that.